pkg/proxy/apis/config/types.go

```diff
@@ -33,6 +33,9 @@ type KubeProxyIPTablesConfiguration struct {
 	MasqueradeBit *int32
 	// masqueradeAll tells kube-proxy to SNAT everything if using the pure iptables proxy mode.
 	MasqueradeAll bool
+	// LocalhostNodePorts tells kube-proxy whether disables the legacy behavior of allowing
```

Comment:
Phrase in the affirmatrive:

 LocalhostNodePorts tells kube-proxy to allow service NodePorts to be accessed via localhost (iptables mode only).